docs/c_cpp_properties.json

```diff
@@ -0,0 +1,113 @@
+            ],
+            "includePath": [
+                "include",
+                "include/libc",
+                "src",
+                "build/gc-us",
+                ".",
+                "extracted/gc-us"
+            ],
+            "defines": [
+                "_LANGUAGE_C",
+                "OOT_VERSION=OOT_GC_US",
+                "OOT_REGION=REGION_US",
+                "OOT_NTSC=1",
+                "OOT_MQ=0",
+                "OOT_DEBUG=0",
+                "NDEBUG",
+                "F3DEX_GBI_2",
+                "F3DEX_GBI_PL",
+                "GBI_DOWHILE"
+            ],
+            "cStandard": "gnu89"
+        },
+        {
+            "name": "oot-gc-eu",
+            "compilerArgs": [
+                "-m32"
+            ],
+            "includePath": [
+                "include",
+                "include/libc",
+                "src",
+                "build/gc-eu",
+                ".",
+                "extracted/gc-eu"
+            ],
+            "defines": [
+                "_LANGUAGE_C",
+                "OOT_VERSION=OOT_GC_EU",
+                "OOT_REGION=REGION_EU",
+                "OOT_PAL=1",
+                "OOT_MQ=0",
+                "OOT_DEBUG=0",
+                "NDEBUG",
+                "F3DEX_GBI_2",
+                "F3DEX_GBI_PL",
+                "GBI_DOWHILE"
+            ],
+            "cStandard": "gnu89"
+        },
+        {
+            "name": "oot-gc-eu-mq",
+            "compilerArgs": [
+                "-m32"
+            ],
+            "includePath": [
+                "include",
+                "include/libc",
+                "src",
+                "build/gc-eu-mq",
+                ".",
+                "extracted/gc-eu-mq"
+            ],
+            "defines": [
+                "_LANGUAGE_C",
+                "OOT_VERSION=OOT_GC_EU_MQ",
+                "OOT_REGION=REGION_EU",
+                "OOT_PAL=1",
+                "OOT_MQ=1",
+                "OOT_DEBUG=0",
+                "NDEBUG",
+                "F3DEX_GBI_2",
+                "F3DEX_GBI_PL",
+                "GBI_DOWHILE"
+            ],
+            "cStandard": "gnu89"
+        },
+        {
+            "name": "oot-gc-eu-mq-dbg",
+            "compilerArgs": [
+                "-m32"
+            ],
+            "includePath": [
+                "include",
+                "include/libc",
+                "src",
+                "build/gc-eu-mq-dbg",
+                ".",
+                "extracted/gc-eu-mq-dbg"
+            ],
+            "defines": [
+                "_LANGUAGE_C",
+                "OOT_VERSION=OOT_GC_EU_MQ_DBG",
+                "OOT_REGION=REGION_EU",
+                "OOT_PAL=1",
+                "OOT_MQ=1",
+                "OOT_DEBUG=1",
+                "F3DEX_GBI_2",
+                "F3DEX_GBI_PL",
+                "GBI_DOWHILE",
+                "GBI_DEBUG"
+            ],
+            "cStandard": "gnu89"
+        }
+    ],
+    "version": 4
+}
```

Comment:
add a newline?

Reply:
I can, the vscode json formatter removes it

Reply:
Huh, for me it doesn't remove it (also VS Code) but it's not important